lte/gateway/configs/gateway.mconfig

```diff
@@ -159,7 +159,21 @@
         "dsn_python": "",
         "dsn_native": "",
         "upload_mme_log": false,
-        "sample_rate": 1.0
+        "sample_rate": 1.0,
+        "exclusion_patterns": [
+          "GetServiceInfo",
```

Comment:
This value should match the default value from the API northbound side. (I'm not sure if there is a way to specify a default value here, though).

If the on-device gateway mconfig value differs from the default cloud streamed version, that would lead to an unnecessary restart of all services.

Reply:
I've added it as a default in swagger and confirmed that the default list is used if the PUT method is called without the exclusion list.